Makefile

```diff
@@ -39,6 +39,12 @@ docker-image: ## Build the docker image
 docker-start: docker-image ## Start a local development container
 	docker run --rm -it -v $(CURDIR)/content:/librarium/content/ -p 9000:9000 $(IMAGE)
 
+docker-ci: docker-image ## Start a local development container
+	docker run --rm -it -v $(CURDIR):/librarium $(IMAGE) npm ci
+
+docker-install: docker-image ## Start a local development container
+	docker run --rm -it -v $(CURDIR):/librarium -p 9000:9000 $(IMAGE) npm install
+
```

Comment:
@luthermonson this is not needed as the container is now built with all required nodejs dependencies.

Reply:
it is needed if i ever want to update the npm dependencies myself. it's just a nice to have and this is easily ran by hand if necessary. i can drop it if you dont want it

Reply:
@luthermonson Yeah, please drop. We can control the dependencies and we would reject any changes that include changes to the dependencies.